content/millennium/r4/financial/charge-item.md

```diff
@@ -83,13 +83,18 @@ Search for ChargeItems that meet supplied query parameters:
 
 ### Parameters
 
- Name                  | Required?  | Type          | Description
------------------------|------------|---------------|-------------------------------------------------------
- `_id`                 | Yes        | [`token`]     | The logical resource id associated with the ChargeItem.
+ Name                  | Required?       | Type          | Description
+-----------------------|-----------------|---------------|-------------------------------------------------------
+ `_id`                 | This or context | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | This or _id     | [`token`]     | Encounter associated with event.
+[`_count`]             | No              | [`number`]    | The maximum number of results to return.
+  `-status`            | No              | [`token`]     | The status of the encounter. Example: `billable`
 
 Notes:
 
 - When searching by `_id`, only a single ChargeItem id can be provided.
+- `_count` and `-status` must be provided with context.
```

Comment:
This seems to imply that _count is a required parameter when context is passed. We can probably just remove _count from this line.

If we feel it's necessary to specify where _count can be used we could separate it into a line specifying `'_count' cannot be provided with '_id'`

Reply:
+1 to removing _count from this. I'd say we only add the other note for _count if we are doing it in other resources.

Reply:
removed _count here https://github.com/cerner/fhir.cerner.com/pull/665/commits/66cb49e49a70cc2e124fb6348991af2240402aa6#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR96